### DISCUSSION TITLE
Link to `Rook has deprecated CSI network "holder" pods.` is broken/wrong

### DISCUSSION THREAD

**Discussion by a1994sc** [rook/rook#14625] [General]
the link in the `1.15.0` release goes to a 404

I believe the correct link is the following:

https://github.com/rook/rook/blob/master/Documentation/CRDs/Cluster/network-providers.md#holder-pod-deprecation

**Comment by travisn** [comment]
Ok fixed it now thanks!

**Reply by a1994sc** [reply]
No problem!
